### PR TITLE
Update fonts to Montserrat, add button transitions

### PR DESCRIPTION
**Pull request by kartikch918**
Add new font Montserrat (Spotify style font)
https://fonts.google.com/specimen/Montserrat

### CODE REVIEW

**Comment by saadpasta** [comment]
@kartikch918 Can You Show Some Screenshots

**Comment by kartikch918** [comment]
New font preview:
Screenshot (51)

File: src/components/button/Button.css [hunk 1]
```diff
@@ -14,7 +15,11 @@
   font-size: 16px;
   cursor: pointer;
 }
-
+.main-button:hover {
+  background-color: #ffffff;
+  color: black;
+  transition: ease-in 0.3s;
+}
```
Comment (saadpasta) [inline src/components/button/Button.css]: Do you think this hover effect is necessary

Comment (kartikch918) [reply src/components/button/Button.css]: In my opinion, yes. Some simple lightweight transitions can make a page feel a lot smoother and interactive. Also it fits in with the theme too but it's your repo so it's your call mate.
Remove the commit if you don't want to add it.

Comment (saadpasta) [reply src/components/button/Button.css]: Bro's it's your Portfolio also it's your choice also and i think you made your point clear and I love it 
Thanks For Contributing

**Pull request merged**